vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -273,16 +273,30 @@ License Setup
 
 Create a Kubernetes `Secret` from the License Key provided by Loft in the `Namespace` where you are installing the air-gapped vCluster instance:
 
+:::tip
+License secret is already `base64` encoded.
+:::
+
 ```bash title="Create Kubernetes Secret"
-# Create license key file
-cat <<EOF > vcluster-pro-air-gapped-license.txt
-YOUR_LICENSE_KEY_HERE
-EOF
+#!/bin/bash
+
+# Set the license key as an environment variable (already base64 encoded)
+export VCLUSTER_LICENSE_KEY="YOUR_BASE64_ENCODED_LICENSE_KEY_HERE"
 
-# Create namespace and secret
+# Optionally create the namespace
 kubectl create namespace vcluster-ns
-kubectl create secret generic vcluster-platform-api-key -n vcluster-ns \
-    --from-file=license-key=./vcluster-pro-air-gapped-license.txt
+
+# Create secret
+kubectl create -f - <<EOF
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vcluster-platform-api-key
+  namespace: vcluster-ns
+type: Opaque
+data:
+  license-key: ${VCLUSTER_LICENSE_KEY}
```

Comment:
This key would work if it's the only key in the `data:` map. This is how we parse license key currently: https://github.com/loft-sh/vcluster-pro/blob/main/pkg/license/online/platform/connection.go#L68-L76
maybe let's change the key to `license` here?

Reply:
@kmadel since this is based on the Notion doc, can you help clarify? It looks like `license` would be the right key indeed.